README.md

```diff
@@ -12,18 +14,22 @@ To install `cmd-ai`, use the following command:
 npm install -g cmd-ai
 ```
 
-Ensure you have Node.js installed on your system before proceeding with the installation.
+Ensure you have Node.js installed (version 18 or higher recommended) on your system before proceeding with the installation. The first time you use the local model, it will automatically download the model files (approximately 300MB for the model and 2gb for the ONNX data), which may take some time.
```

Comment:
It's not by default. The user actually have to select it the first time uses the command (while running `ai config`). I would change it